components/suggestion/suggestion_list.d.ts

```diff
@@ -1,43 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Would you mind re-adding this file? We need it to type the regular SuggestionList still since this PR only adds types to the SearchSuggestionList

Reply:
Yeah, but when i am importing suggestion_list.jsx in a component, this file is getting imported insted.
Maybe we should change **suggestion_list.d.ts**'s name to something else.
or is there any other way to work around this?

Reply:
That seems like what we'd want here. Since it's a `.d.ts` file specifically, TypeScript will know to use the definitions there for type checking and the actual code for compilation. Without that file, TS can't determine the type of `SuggestionList` until we migrate `suggestion_list.jsx` to TypeScript as well, and I think there's a separate PR in the works to do that.